### PR TITLE
tasks: Add user-research.md.

### PR DESCRIPTION
**Pull request by rishig**
Thoughts?

### CODE REVIEW

**Comment by rishig** [comment]
+ @trueskawka, if you have time for a review :).

File: tasks/2017/user-research.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+* Get ready to write or type. There will be a lot to note down, and it will
+  happen fast :).
+* Send them to zulipchat.com, and ask them to create an account. The
+  organization can be called something like test-[their name], and they can
+  use a fake name; that part is not important.
+
+If the screen is being recorded, you only have to jot down
+* The things they say, and when they say them.
+* Anything they seem confused by.
+* A link to the video. You can share videos privately on youtube.
```
Comment (lonerz) [inline tasks/2017/user-research.md]: r/youtube/YouTube

File: tasks/2017/user-research.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+If not, also jot down:
+* Any bugs they run in to.
```
Comment (lonerz) [inline tasks/2017/user-research.md]: r/in to/into?

**Comment by rishig** [comment]
@lonerz Good catch with the "in to" to "into" :). Re-pushed with both issues fixed.

**Pull request merged**